examples/ui/scroll.rs

```diff
@@ -345,3 +363,11 @@ pub fn update_scroll_position(
         }
     }
 }
+
+fn report(sq: Query<Ref<ComputedNode>, With<ScrollingList>>) {
+    for node in sq {
+        println!("node size = {} ", node.size());
+        println!("content size = {} ", node.content_size());
+        println!("scrollbar size = {}", node.scrollbar_size);
+    }
+}
```

Comment:
This looks like a leftover from debugging

Reply:
Oops, yes that would be correct.